zoltar-tensorflow/src/main/java/com/spotify/zoltar/tf/JTensor.java

```diff
@@ -0,0 +1,82 @@
+/*-
+ * -\-\-
+ * zoltar-tensorflow
+ * --
+ * Copyright (C) 2016 - 2018 Spotify AB
+ * --
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -/-/-
+ */
+
+package com.spotify.zoltar.tf;
+
+import com.google.auto.value.AutoValue;
+import org.tensorflow.DataType;
+import org.tensorflow.Tensor;
+
+import javax.annotation.Nullable;
+import java.nio.DoubleBuffer;
+import java.nio.FloatBuffer;
+import java.nio.IntBuffer;
+import java.nio.LongBuffer;
+
+@AutoValue
+public abstract class JTensor {
+  static JTensor create(Tensor<?> tensor) {
```

Comment:
do you have to make this method `public`?

Reply:
It can be, outside of `Utils#runAndExtract`. I'll make it public?

Reply:
well - it's a question - i'm not sure how useful this is gonna be to make it public, technically ppl should just go for `Utils#runAndExtract`

Reply:
i think we should keep `protected` for now.